kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -353,5 +354,63 @@
 
   .col-name, .col-username
     width: 45%
+    
+  .results-text
+    font-size: 0.9375rem
+    
+  .toolbar
+    margin-top: 30px
+    
+  .toolbar:after
+    content: ''
+    display: table
+    clear: both
+    
+  input[type='search']
+    display: inline-block
+    box-sizing: border-box
+    position: relative
+    top: 0
+    left: 10px
+    height: 100%
+    width: 85%
+    border-color: transparent
+    background-color: transparent
+    clear: both
+    
+  .search-box .icon
+    display: inline-block
+    float: left
+    position: relative
+    fill: $core-text-annotation
+    left: 5px
+    top: 5px
+
+  .search-box
+    border-radius: 5px
+    padding: inherit
+    border: 1px solid #c0c0c0
+    max-width: 400px
+    height: $toolbar-height
+    float: left
+    
+  @media screen and (min-width: $portrait-breakpoint + 1)
```

Comment:
we're trying to avoid media queries and transition to the new `responsive-mixin`. let me know if you'd like help with this